test/models/user_test.rb

```diff
@@ -3,7 +3,51 @@
 require 'test_helper'
 
 class UserTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  setup do
+    @alice = users(:alice)
+    @bob = users(:bob)
+    @carol = users(:carol)
+    @no_name = users(:no_name)
+  end
+
+  test '#following?(user)' do
+    assert @alice.following?(@bob)
+    assert_not @bob.following?(@carol)
+    assert_not @carol.following?(@carol)
+  end
+
+  test '#followed_by?(user)' do
+    assert @bob.followed_by?(@alice)
+    assert_not @bob.followed_by?(@carol)
+    assert_not @carol.followed_by?(@carol)
+  end
+
+  test '#follow(user)' do
+    assert_not @bob.following?(@carol)
+    assert_difference('Relationship.count', 1) do
```

Comment:
```suggestion
    assert_difference(@bob.following?(@carol)) do
```

みたいに書いて、未フォローからフォロー中に変わった、というテストを表現できないですかね〜？（できるかどうかは未確認）

できない場合、フォローしたあとに`assert @bob.following?(@carol)`を検証しておくとさらに良いかもしれません。
（全然別のユーザーが対象になっていないことを確認する意味で）

Reply:
なるほどー。 [assert_changes](https://api.rubyonrails.org/v7.0.3/classes/ActiveSupport/Testing/Assertions.html#method-i-assert_changes) でテストの意図を表現できそうです！